src/middleware.ts

```diff
@@ -70,6 +70,13 @@ export const withWebhookSignatureVerified =
     const { [SALEOR_DOMAIN_HEADER]: saleorDomain, "saleor-signature": payloadSignature } =
       request.headers;
 
+    if (!payloadSignature) {
+      return Response.BadRequest({
+        success: false,
+        message: "Missing payload signature.",
```

Comment:
here you have `payload signature`, below we just refer that that entity as a `signature`. We should stick to one.